pkg/envoy/testdata/envoy_bootstrap_v2.tmpl.json

```diff
@@ -48,6 +48,7 @@
                   "codec_type": "AUTO",
                   "stat_prefix": "stats",
                   "route_config": {
+        　　　　　　　"name": "static-prometheus:{{ .listenerPort }}",
```

Comment:
is there a tab here or something? Github is rendering this weirdly

Reply:
fixed, It might add by my IDE